### PR TITLE
To bagit

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: data/kant_aufklaerung_1784/url-sources.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+https://github.com/OCR-D/assets/raw/master/data/kant_aufklaerung_1784/kant_aufklaerung_1784_0017.tif data/kant_aufklaerung_1784_0017.tif
+https://github.com/OCR-D/assets/raw/master/data/kant_aufklaerung_1784/kant_aufklaerung_1784_0020.tif data/kant_aufklaerung_1784_0020.tif
```
Comment (VolkerHartmann) [inline data/kant_aufklaerung_1784/url-sources.txt]: Are these URLs correct?
The files will be moved during this pull request to https://github.com/OCR-D/assets/raw/master/data/kant_aufklaerung_1784/data/kant_aufk....., won't they?

**Comment by kba** [comment]
> Why not using fetch.txt instead of url-sources.txt which should be supported by the libraries?

Because it has a different purpose. `fetch.txt` is for downloading URL to files in `/data`. `url-sources` (proposals for different naming welcome) is for specifying aliases of URLs to local files. See https://github.com/OCR-D/core/issues/176

File: data/kant_aufklaerung_1784/url-sources.txt (hunk 1 above)
Comment (kba) [reply data/kant_aufklaerung_1784/url-sources.txt]: Deleted url-sources

**Comment by kba** [comment]
Calculated by hand, apparently wrong, fixing.

**Pull request merged**